### ISSUE TITLE
Unable to see the menu bar on Ubuntu 19.04

### ISSUE THREAD

**Issue by v-xuanzh** [microsoft/AzureStorageExplorer#1503]
**Storage Explorer Version:** 1.9.0_**rel/1.9.0**_20190625.3
**Platform/OS:** Linux Ubuntu 19.04
**Architecture:** x64
**Regression From:** Not a regression

**Steps to reproduce:**
1. Launch Storage Explorer.
2. Try to check the menu bar.

**Expect Experience:**

Able to see the menu bar.

**Actual Experience:**

Unable to see the menu bar.
![image](https://user-images.githubusercontent.com/34729022/60091964-605ba800-9778-11e9-829f-6db282223101.png)

**More info:**

This issue doesn't reproduce on Ubuntu 18.04/16.04/14.04

**Comment by craxal** [comment]
@v-xuanzh Was this run straight from the uncompressed .tar.gz file or was this installed from a snap?

**Comment by jinglouMSFT** [comment]
@craxal if i remembered it correctly, you have installed from snap on 19.04 and everything worked ok?

**Comment by craxal** [comment]
I haven't testing anything on Ubuntu 19.04. The latest I've tested on is 18.04.

**Comment by craxal** [comment]
Spun up my own Ubuntu 19 machine, and I'm able to reproduce from a snap. Really weird. If you look very closely, you can see the menus are still there. Just in a slightly different color than the window's title bar. If you switch to high contrast, you can see the menus just fine.

Did a little research, and it sounds like we can fix this if we update to Electron 4.0.8 or later.

**Comment by v-xuanzh** [comment]
Hi @craxal ,Today we try to install the latest real signed build **20190626.4** using the snap on Linux, but we got build is **20190626.3** actually.

**Steps to reproduce:**

1. Make sure Ubuntu 19.04 have supported snap.
2. Download the 'StorageExplorer_latest_amd64.snap' from Linux Artifactes(20190626.4)
3. Run the following commands:

`sudo snap install StorageExplorer_latest_amd64.snap --dangerous`
`snap connect storage-explorer:password-manager-service :password-manager-service`

4. Launch Storage Explorer by the command: `storage-explorer`
5. Check the build version after launching Storage Explorer successfully.

**Expect:**

The shown build is **20190626.4**

**Actual:**

The shown build is **20190626.3**
![image](https://user-images.githubusercontent.com/34729022/60145616-30081e00-97f9-11e9-8188-326b53d7b967.png)

**Comment by v-xuanzh** [comment]
Hi @craxal we installed Storage Explorer using .tar.gz file.

**Comment by craxal** [comment]
Confirmed with Electron 4.2.5. Upgrading Electron will fix this issue.

**Comment by v-xuanzh** [comment]
This issue has been fixed on latest master build 20190711.5. So closed it.
